system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -409,7 +410,7 @@ parameter_types! {
 	// Fellows pluralistic body.
 	pub const FellowsBodyId: BodyId = BodyId::Technical;
 	/// The asset ID for the asset that we use to pay for message delivery fees.
-	pub FeeAssetId: AssetId = AssetId(xcm_config::KsmRelayLocation::get());
+	pub FeeAssetId: AssetId = AssetId(RelayChainLocation::get());
```

Comment:
Will this also work after the migration when AH is the reserve for KSM? OR are both RC and AH reserves?